src/main/java/mate/academy/spring/config/AppConfig.java

```diff
@@ -13,10 +14,9 @@
 
 @Configuration
 @PropertySource("classpath:db.properties")
-@ComponentScan(basePackages = {
-        "mate.academy.spring",
-})
+@ComponentScan(basePackages = {"mate.academy.spring"})
 public class AppConfig {
+    @Autowired
```

Comment:
we can remove annotation above `Environment env`, because we use injection via constructor